test/ExecutionFramework.cpp

```diff
@@ -50,25 +55,294 @@ string getIPCSocketPath()
 	return ipcPath;
 }
 
+evmc::vm& getVM()
+{
+	static unique_ptr<evmc::vm> theVM;
```

Comment:
You can just do `static evmc::vm theVM = ...`;

Reply:
but then I cannot try different locations for libevmone.so